File: src/main/java/tax/bilibili/nineblog/entity/Article.java
package tax.bilibili.nineblog.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Article
 *
 * @author sheip9
 * @since 2023/10/22 19:31
 */
@Data
@NoArgsConstructor
@TableName("articles")
public class Article {
    @TableId(value = "id", type = IdType.AUTO)
    Long id;
    Long author;
    String content;
    Long createDate;
    Long lastEdit;
}

File: src/main/java/tax/bilibili/nineblog/entity/Article.kt
package tax.bilibili.nineblog.entity

import com.mybatisflex.annotation.Id
import com.mybatisflex.annotation.KeyType
import com.mybatisflex.annotation.Table

@Table("articles")
data class Article(
    @Id(keyType = KeyType.Auto)
    val id: Int,
    val authorId: Int,
    val title: String,
    val content: String,
    val createdAt: String,
    val updatedAt: String,
    val visibilityLevel: Int,
)
